tests/p_grub2/01_grub2_secureboot_signed.sh

```diff
@@ -5,9 +5,15 @@ t_Log "Running $0 -  Verifying that grub2-efi is correctly signed with correct c
 
 arch=$(uname -m)
 
+signing_key='CentOS Secure Boot Signing 202'
```

Comment:
Can we factor these into their own vars in 0_lib?

Reply:
Do we really should put it to 0_lib?
I used it only for reusable vars, but this `signing_key` var is used only in this file